packages/esm-patient-chart-app/src/visit/visit-form/visit-form.workspace.tsx

```diff
@@ -739,22 +751,22 @@ const StartVisitForm: React.FC<StartVisitFormProps> = ({
                     <BaseVisitType visitTypes={allVisitTypes} />
                   )}
                 </div>
-              </section>
-            )}
 
-            {errors?.visitType && (
-              <section>
-                <div className={styles.sectionTitle}></div>
-                <div className={styles.sectionField}>
-                  <InlineNotification
-                    role="alert"
-                    style={{ margin: '0', minWidth: '100%' }}
-                    kind="error"
-                    lowContrast={true}
-                    title={t('missingVisitType', 'Missing visit type')}
-                    subtitle={t('selectVisitType', 'Please select a Visit Type')}
-                  />
-                </div>
+                {errors?.visitType && (
+                  <section>
+                    <div className={styles.sectionTitle}></div>
+                    <div className={styles.sectionField}>
+                      <InlineNotification
+                        role="alert"
+                        style={{ margin: '0', minWidth: '100%' }}
+                        kind="error"
+                        lowContrast={true}
+                        title={t('missingVisitType', 'Missing visit type')}
+                        subtitle={t('selectVisitType', 'Please select a Visit Type')}
+                      />
+                    </div>
+                  </section>
+                )}
```

Comment:
Moves the inline notification higher up so it looks more closely connected to the LocationSelector component above it:

![CleanShot 2025-01-11 at 2  02 56@2x](https://github.com/user-attachments/assets/7d217b00-7088-456e-84ed-e8a63cc780c3)